python/private/proto/py_proto_library.bzl

```diff
@@ -109,6 +107,7 @@ def _py_proto_aspect_impl(target, ctx):
     return [
         _PyProtoInfo(
             imports = depset(
+                [proto_root],
```

Comment:
Could you explain in a comment why this is needed, please. Do we have a test for this?

Reply:
Yes, that's why I changed the examples/py_proto_library